### DISCUSSION TITLE
cross platform components

### DISCUSSION THREAD

**Discussion by surya239** [slint-ui/slint#6846] [Q&A]
I am trying to build cross platform reusable UI component with the help of Slint and embed this in native application built for macOS , Windows , Android , Linux and iOS . Is there a possibility to achieve the same ?

**Comment by szecket** [comment]
Tell me a bit more about what you're looking to do?  what kind of components? Are you asking if it is possible to get information about the platform or if it possible to create different views for different platforms or if Slint can be reused across platforms?
